src/nav-soknad/components/linkButton/LinkButton.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import * as classNames from "classnames";
+import styled from "styled-components";
+
+const StyledLinkButton = styled.button`
+    border-style: none;
+    cursor: pointer;
+    padding: 0;
+    text-decoration: underline;
```

Comment:
trenger ikke gjøres, men kan eventuelt bytte underline med border-bottom for å få litt mer space :)

Reply:
Hmm.. Ser ut som at `text-decoration: underline` blir satt i `lenke` klassen også, så kan uansett fjerne den herfra. :)